cg/services/illumina_services/illumina_post_processing_service/illumina_post_processing_service.py

```diff
@@ -183,6 +183,7 @@ def post_process_illumina_flow_cell(
         if self.dry_run:
             LOG.info(f"Dry run: will not post-process Illumina run {sequencing_run_name}")
             return
+        self.delete_sequencing_run_data(flow_cell_id=run_directory_data.id)
```

Comment:
as this raises an error, should this be inside the try-except?

Reply:
The try and exept for the raised error is inside the function.

Reply:
ok, I will resolve this